index.html

```diff
@@ -11,9 +11,20 @@
 <body>
   <div class="container">
     <!--Toolbar Begins-->
+    <div class="mobile-menu">
+      <div class="mobile-menu-close-toggle-container">
+        <a id="mobile-menu-close-toggle" onclick="toggleClasses()">
+          <img src="./images/icon_cancel.svg" alt="" class="mobile-menu-close-toggle-image">
+        </a>
+      </div>
+      <div class="mobile-menu-links">
+        <a href="#works" class="mobile-menu-links-items" onclick="toggleClasses()">Portfolio</a>
+        <a href="#aboutMe" class="mobile-menu-links-items" onclick="toggleClasses()">About</a><a href="#contact" class="mobile-menu-links-items" onclick="toggleClasses()">Contact</a>
+      </div>
```

Comment:
- [ ] Please ensure that the lines under the links span almost 80% of the mobile menu so it conforms to the Figma design

> Yours
![image](https://user-images.githubusercontent.com/90556221/188859239-a825419a-8839-47c4-8638-ece29154436a.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/188859448-1f711f6a-79da-4a01-a15b-7d870bd1ce19.png)

- [ ] Kindly also ensure that the gap between the links conform to the Figma design

- [ ]  Finally, please make sure that you add the white section at the end of your menu for stylistic conformation to the Figma design.
> Yours
![image](https://user-images.githubusercontent.com/90556221/188859696-2b909342-5240-40b3-a049-da3a3bb764db.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/188859797-9f96bb95-84eb-49dc-95fc-80156551b906.png)

Reply:
Thank you for the suggestions. I have made changes in my code that I hope more closely match your suggestions.